.github/workflows/periodic.yml

```diff
@@ -2,8 +2,10 @@ name: periodic
 
 on:
   schedule:
-    - cron: 45 0,8,16 * * *
-    - cron: 45 4,12,20 * * *
+    - cron: 45 0,8,16 * * 1-5
+    - cron: 45 0 * * 0,6
+    - cron: 45 4,12,20 * * 1-5
+    - cron: 45 12 * * 0,6
```

Comment:
We have a lot of schedules now :)
How about adding a comment explaining the what/why over here

Reply:
Added comments.